app/src/main/kotlin/org/cru/godtools/ui/login/LoginLayout.kt

```diff
@@ -167,3 +172,31 @@ private fun LoginError(error: LoginResponse.Error?, onDismiss: () -> Unit) {
         )
     }
 }
+
+@Composable
+private fun NoInternetError(createAccount: Boolean, onDismiss: () -> Unit) {
+    val context = LocalContext.current
+    val isConnected by context.isConnectedFlow().collectAsState(true)
```

Comment:
this is actually wrong in a subtle manner. every time `NoInternetError` recomposes it will create a new `isConnectedFlow()`. which will cause `collectAsState` to need to start collecting from a new Flow.

You can solve this by wrapping the creation of the flow itself in a `remember {}` block